R/read_extract_district_nursing.R

```diff
@@ -3,9 +3,13 @@
 #' @inherit read_extract_acute
 #'
 #' @export
-read_extract_district_nursing <- function(
-    year,
-    file_path = get_boxi_extract_path(year = year, type = "DN")) {
+read_extract_district_nursing <- function(year,
+                                          file_path = get_boxi_extract_path(year = year, type = "DN")) {
+  # Specify years available for running
+  if (year >= 1516 & year <= 2021) {
+    return(NULL)
+  }
```

Comment:
As with CMH
```suggestion
  if (!(year %in% c("1516", "1617", "1718", "1819", "1920", "2021"))) {
    return(NULL)
  }
```

Reply:
We discussed changing the `read_XXX` extracts to `return(NULL)` when `is.na(file_path)` is supplied as this is now passed through `check_year_valid`. However, do you think we should update this function with this suggestion?

Reply:
Yes, I think only the file path functions should be checking the year, the other ones should check a) is the file path `NA` or b) is the `data` `NULL`.